libs/MeshKernelApi/include/MeshKernelApi/MeshKernel.hpp

```diff
@@ -432,6 +432,13 @@ namespace meshkernelapi
         MKERNEL_API int mkernel_curvilinear_compute_rectangular_grid_on_extension(int meshKernelId,
                                                                                   const meshkernel::MakeGridParameters& makeGridParameters);
 
+        /// @brief Compute a rectangular or circular curvilinear grid
+        /// @param[in] meshKernelId The id of the mesh state
+        /// @param[in] parameters   The structure containing the make grid parameters
+        /// @returns Error code
+        MKERNEL_API int mkernel_curvilinear_compute_circular_grid(int meshKernelId,
```

Comment:
It still bothers me that we use references in a C API... Geen actie nodig.

Reply:
Since when was the keyword *const* added to C?